.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+## Objective
+<!--- Why is this change required? What problem does it solve? -->
+<!--- Provide a general summary of your changes -->
+<!--- If it fixes an open issue, please link to the issue here. -->
+
+## Notes
+<!--- Justify any important decisions made during the development -->
+<!--- Provide documentation or relevant links that justify the use of obscure code when no other alternatives are available -->
+
+## How to test
+<!-- If testing the changes require a special setup or environment, please describe it in detail here -->
+<!--- Provide an unambiguous set of steps to correctly setup and test the changes: -->
+1.
+2.
+3.
+4.
+
+## Breaking changes
+<!--- Does these changes make updating from older versions problematic? -->
+<!--- Is any data lost if updated? -->
+<!--- If so what additional steps should users take to update? -->
+
+## Checklist
+<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
+<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+- [ ] I have read the **[CONTRIBUTING]** document.
+- [ ] I have created a _Codelog changefile_ with the changes made to the branch.
+- [ ] I have created a test proving that my feature/fix does what it intends to do.
+- [ ] I have updated the documentation accordingly. (If needed)
+
+[CONTRIBUTING]: https://github.com/codus/codelog/blob/master/CONTRIBUTING.md
```

Comment:
@luisbevilacqua     
What about moving this link to the checklist item in line 26?